exchanges/exmo/exmo_wrapper.go

```diff
@@ -212,19 +225,20 @@ func (e *EXMO) UpdateTickers(ctx context.Context, a asset.Item) error {
 		return err
 	}
 	for i := range pairs {
-		for j := range result {
-			if !strings.EqualFold(pairs[i].String(), j) {
+		for pair, tick := range result {
+			if !strings.EqualFold(pairs[i].String(), pair) {
```

Comment:
Given that the pair formatting applied via `GetEnabledPairs` will set `pairs[i]` to upper, you could use `strings.ToUpper(pair)` instead of the more costly `EqualFold`. `result -> pair` will also have upper pair strings already so you could actually just go with `pairs[i].String() != pair`

I've only included this because of the other comment.